server/migrations/2020-06-30-135809_remove_mat_views/up.sql

```diff
@@ -490,6 +491,7 @@ select
 	u.actor_id as creator_actor_id,
 	u.local as creator_local,
 	u.name as creator_name,
+  u.published as creator_published,
```

Comment:
You can't just edit an existing migration, you need to create a new one with `diesel migration generate add_creator_published`

And run `diesel migration run` and `diesel migration revert` to test your changes.

If you need help w/ a migration, let me know because they aren't easy to do bc of the table complexity.

Reply:
Alright, I'm gonna try to do this next, will let you know.

Reply:
Alright, it took me some time, but I think I managed to add migration with the new field `creator_published`.

When you have some time you can check it here 7c35fc546bb148047087db6ed95b1579bab08238.